tests/providers/aws/services/backup/backup_service_test.py

```diff
@@ -0,0 +1,136 @@
+from datetime import datetime
+from unittest.mock import patch
+
+import botocore
+
+from prowler.providers.aws.lib.audit_info.audit_info import current_audit_info
```

Comment:
Please, mock the `audit_info` object instead of import it in order to avoid race conditions